src/core/include/openvino/op/util/attr_types.hpp

```diff
@@ -20,6 +20,12 @@ enum class PadMode { CONSTANT = 0, EDGE, REFLECT, SYMMETRIC };
 OPENVINO_API
 std::ostream& operator<<(std::ostream& s, const PadMode& type);
 
+/// \brief Fill modes for the `SegmentMax` operator.
```

Comment:
If this is specific for SegmentMax op only, then it should be placed in the op dedicated namespace and file.
Otherwise generalize the comment.

Reply:
There is a chance to extend it and reuse it with other ops in the future like SegmentMin 
``suggestion
/// \brief Fill modes to set default value for operators like `SegmentMax`.
```
But if it's preferred to use a separate enums for such ops, it should be inside the SegmentMax class.